src/autocomplete/typeahead.js

```diff
@@ -31,12 +31,22 @@ function Typeahead(o) {
   this.autoselectOnBlur = !!o.autoselectOnBlur;
   this.openOnFocus = !!o.openOnFocus;
   this.minLength = _.isNumber(o.minLength) ? o.minLength : 1;
+
+  o.hint = !!o.hint;
+
+  if (o.hint && o.appendTo) {
+    throw new Error('[autocomplete.js] hint and appendTo options can\'t be used at the same time');
```

Comment:
^ What he said, clarified a bit more in my last comment (`appendTo` vs `dropdownMenuContainer`).